dashboard/test/controllers/api/v1/pd/applications_controller_test.rb

```diff
@@ -673,7 +673,9 @@ class ApplicationsControllerTest < ::ActionController::TestCase
     end
 
     test 'cohort csv download returns expected columns for teachers' do
-      create TEACHER_APPLICATION_FACTORY, :locked, course: 'csd'
+      application = create TEACHER_APPLICATION_FACTORY, course: 'csd'
+      application.status = 'accepted_not_notified'
```

Comment:
Prefer application.update(status: 'accepted_not_notified'). Also do we have a const defining this string?